backend/test/images/test-run.sh

```
@@ -240,5 +256,8 @@ go run github.com/onsi/ginkgo/v2/ginkgo -r -v -p \
   -apiUrl="$API_URL" \
   -authToken="$API_TOKEN" \
   -disableTlsCheck=true \
+  -customPipIndexURL=$CUSTOM_PIP_INDEX_URL \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)